mutation_compactor.hh

```diff
@@ -293,7 +293,6 @@ public:
         , _collector(std::make_unique<mutation_compactor_garbage_collector>(_schema))
     {
         static_assert(sstable_compaction(), "This constructor can only be used for sstable compaction.");
-        static_assert(!only_live(), "SSTable compaction cannot be run with emit_only_live_rows::yes.");
```

Comment:
You can remove the function only_live() too, and also the comment above the class referring to only_live.

Reply:
This is done in a later patch. I forgot about the comment though.

Reply:
Done in v2.